cpp/src/IceSSL/SChannelTransceiverI.cpp

```diff
@@ -956,23 +620,27 @@ SChannel::TransceiverI::closing(bool initiator, exception_ptr)
 void
 SChannel::TransceiverI::close()
 {
-    if (_sslInitialized)
+    if (_ssl.dwLower || _ssl.dwUpper)
     {
         DeleteSecurityContext(&_ssl);
-        _sslInitialized = false;
+        _ssl = {0, 0};
```

Comment:
Ditto for this and _credentials below (should they be `nullptr`)

Reply:
these aren't pointers